extensions/amp-subscriptions/0.1/platform-store.js

```diff
@@ -80,16 +83,38 @@ export class PlatformStore {
    */
   resolvePlatform(serviceId, platform) {
     this.subscriptionPlatforms_[serviceId] = platform;
+    this.onPlatformResolvedCallbacks_.fire({
+      serviceId,
+    });
+  }
+
+  /**
+   * Returns a promise to be solved when a platform is resolved.
+   * @param {string} serviceId
```

Comment:
`return` is missing. But please see the comments above.

Reply:
Fixed